integration_tests/test_ibc_timeout.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+
+from .ibc_utils import RATIO, assert_ready, get_balance, prepare, prepare_network
+from .utils import ADDRS, eth_to_bech32, wait_for_fn
+
+
+@pytest.fixture(scope="module")
+def ibc(request, tmp_path_factory):
+    "start-cronos"
+    name = "ibc_timeout"
+    path = tmp_path_factory.mktemp(name)
+    network = prepare_network(path, name)
+    try:
+        yield next(network)
+    finally:
+        try:
+            next(network)
+        except StopIteration:
+            pass
+
+
+def test_ibc(ibc):
+    src_amount = prepare(ibc)
+    dst_amount = src_amount * RATIO  # the decimal places difference
+    dst_denom = "basetcro"
+    dst_addr = eth_to_bech32(ADDRS["signer2"])
+    old_dst_balance = get_balance(ibc.cronos, dst_addr, dst_denom)
+
+    new_dst_balance = 0
+
+    def check_balance_change():
+        nonlocal new_dst_balance
+        new_dst_balance = get_balance(ibc.cronos, dst_addr, dst_denom)
+        return new_dst_balance != old_dst_balance
+
+    wait_for_fn("balance change", check_balance_change)
+    assert old_dst_balance + dst_amount == new_dst_balance
+
+
+def test_cronos_transfer_tokens(ibc):
```

Comment:
Maybe rename the test to test_cronos_transfer_timeout so that it is more clear in the test reporting